eve/main.yml

```diff
@@ -47,7 +56,7 @@ models:
   - SetProperties: &set_operator_properties
       properties:
         OPERATOR_IMAGE_NAME: registry.scality.com/sf-eng/zenko-operator
-        OPERATOR_IMAGE_TAG: 004087b
+        OPERATOR_IMAGE_TAG: '3879889'
```

Comment:
We may want to pull this from deps.txt or the one source of truth later. Having to update this tag when the operator is updated will be hard.

Reply:
I had thought of using the floating tag. Will probably discuss this in the `Zenko: how do we test?` meeting next Tuesday

Reply:
The operator version that runs should be the exact one that would be embedded in the ISO if we built it directly from this exact tree. That version is available in the `solution/deps.txt` file. Otherwise the tests don't really reflect what would be built.